conf/common.env

```diff
@@ -11,3 +10,17 @@ RETRY_INTERVAL=5
 LOGGING_TYPE=DEVEL
 LOGGING_LEVEL_APP=INFO
 LOGGING_LEVEL_LIBS=WARNING
+
+DB_TYPE=postgres
+DB_HOST=db
+DB_NAME=app
+DB_USER=admin
+DB_PASSWD=passwd
+DB_PORT=5432
+
+LISTENER_KAFKA_TOPIC=host.packages
+LISTENER_KAFKA_ADDRESS=platform:9092
+LISTENER_BUFFER_SIZE=10
+LISTENER_BATCH_WRITE=off
+
+BENCHMARK_MESSAGES=50
```

Comment:
As you increased msgs number from 30->50 I got this, did you too?
~~~
platform_1  | >>>>[2019-11-07 10:48:11,939] WARN [Producer clientId=console-producer] Got error produce response with correlation id 4 on topic-partition host.packages-0, retrying (2 attempts left). Error: NETWORK_EXCEPTION (org.apache.kafka.clients.producer.internals.Sender)
~~~
I've got this 4x.

Reply:
Yes, but the tests seem to be working, I'm rewriting the sending script to use single producer

Reply:
Not quite, it sends some messages multiple times. It's seen from items in database (<50). Set it back to 30, please. It's working without warns and these mentioned problems.

Reply:
Try now. I added python script to send the messages, so that the messages are sent over single producer.